app/components/primer/button_component.rb

```diff
@@ -111,5 +119,10 @@ def initialize(
     def link?
       @scheme == LINK_SCHEME
     end
+
+    def trimmed_content
+      # strip unsets `html_safe`, so we have to set it back again to guarantee that HTML blocks won't break
+      content.strip.html_safe # rubocop:disable Rails/OutputSafety
```

Comment:
opinions here? we treat `content` as safe, so I don't think this should be a problem

Reply:
Can we add a check to make sure `content` is safe?